ansible/roles_ocp_workloads/ocp4_workload_nginxplus/tasks/workload.yml

```diff
@@ -15,10 +15,13 @@
 
 - name: create secret for quay.io
   # yamllint disable-line rule:line-length
-  shell: oc create secret docker-registry quay-login --docker-username="{{ quay_username }}" --docker-server="{{ private_registry_url }}" --docker-password="{{ quay_password }}" -n "{{ nginx_namespace }}"
+  shell: oc create secret docker-registry quay-login --docker-username="{{ quay_username }}" --docker-server="{{ private_registry_url }}" --docker-password="{{ quay_password }}" -n "{{ nginx_namespace }}"  || /bin/true
```

Comment:
Hm. Not crazy about the oc command in shell. But given the following import image I guess it's OK. It would be better to use the kubernetes.core.k8s module for this.

Reply:
I wasn't sure how to base64-encod the Docker configuration JSON file to use in the template file.  I would want it generated each time in case I need to change any of the params.  If there is a way to do that, I can change this to a template file.